[2024-04-02T16:41:57.328+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_cron.third_task manual__2024-04-02T16:41:52.858575+00:00 [queued]>
[2024-04-02T16:41:57.337+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_cron.third_task manual__2024-04-02T16:41:52.858575+00:00 [queued]>
[2024-04-02T16:41:57.338+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-02T16:41:57.354+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): third_task> on 2024-04-02 16:41:52.858575+00:00
[2024-04-02T16:41:57.361+0000] {standard_task_runner.py:60} INFO - Started process 848 to run task
[2024-04-02T16:41:57.365+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'first_cron', 'third_task', 'manual__2024-04-02T16:41:52.858575+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/first_crnschedular.py', '--cfg-path', '/tmp/tmp466j2j2q']
[2024-04-02T16:41:57.366+0000] {standard_task_runner.py:88} INFO - Job 182: Subtask third_task
[2024-04-02T16:41:57.455+0000] {task_command.py:423} INFO - Running <TaskInstance: first_cron.third_task manual__2024-04-02T16:41:52.858575+00:00 [running]> on host 5c9eefa8eb82
[2024-04-02T16:41:57.570+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_cron' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T16:41:52.858575+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-02T16:41:52.858575+00:00'
[2024-04-02T16:41:57.573+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-02T16:41:57.575+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is my third task']
[2024-04-02T16:41:57.595+0000] {subprocess.py:86} INFO - Output:
[2024-04-02T16:41:57.597+0000] {subprocess.py:93} INFO - hello world, this is my third task
[2024-04-02T16:41:57.598+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-02T16:41:57.629+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=first_cron, task_id=third_task, execution_date=20240402T164152, start_date=20240402T164157, end_date=20240402T164157
[2024-04-02T16:41:57.658+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-02T16:41:57.679+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
